### PR TITLE
Alignment recipe for CSJ.

### PR DESCRIPTION
**Pull request by jnishi**
Add an alignment recipe for CSJ. Test data is created from the evaluation dataset and is tested in this recipe.

### CODE REVIEW

**Comment by sw005320** [comment]
Many thanks!
Maybe you can use a symlink to point https://github.com/espnet/espnet/tree/master/egs/csj/asr1/local to avoid duplicated files (or a symlink to required files under the local directory?).
I guess egs/csj/align1/local/score_sclite.sh is not used. Please check it.

**Comment by jnishi** [comment]
Unnecessary files are removed and necessary files which are the same as the asr recipe are changed to symlinks.
`local/csj_rm_tag_sp_space.sh ` and `local/csj_eval_data_prep.sh` needs some modification.

Please check.

File: egs/csj/align1/run.sh [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+do_delta=false
+cmvn=
+
+# data
+CSJDATATOP=/export/corpora5/CSJ/USB
+CSJVER=usb                          # see kaldi/egs/csj/s5/run.sh about the version
+
+# exp tag
+tag="" # tag for managing experiments.
+
```
Comment (sw005320) [inline egs/csj/align1/run.sh:31]:
```suggestion
```
According to https://github.com/espnet/espnet/pull/2531/checks?check_run_id=1185652786#step:7:211

**Comment by jnishi** [comment]
Thank you for describing CI's message.
Would you mind advising me to fix codecov's error?

**Comment by sw005320** [comment]
> Thank you for describing CI's message.
> Would you mind advising me to fix codecov's error?

We can ignore it.
It is not easy to include the test coverage for the recipe.

Many thanks!

**Pull request merged**